include/evmc/evmc.h

```diff
@@ -116,27 +119,27 @@ struct evmc_message
     /**
      * The amount of Ether transferred with the message.
      */
-    struct evmc_uint256be value;
+    evmc_uint256be value;
 
     /**
      * The optional value used in new contract address construction.
      *
      * Ignored unless kind is EVMC_CREATE2.
      */
-    struct evmc_uint256be create2_salt;
+    evmc_bytes32 create2_salt;
 };
 
 
 /** The transaction and block data for execution. */
 struct evmc_tx_context
```

Comment:
Maybe typedef this one, too, to write `evmc_tx_context` instead od `struct evmc_tx_context`?